.gitignore

```
@@ -19,4 +19,9 @@ pom.xml.versionsBackup
 *.orig
 *.old
 *.md.html
-*.flattened-pom.xml
\ No newline at end of file
+*.flattened-pom.xml
+/.apt_generated/
+/.apt_generated_tests/
+
+/.settings/org.eclipse.core.resources.prefs
+/.settings/org.eclipse.m2e.core.prefs
```

```suggestion
/.apt_generated_tests/
```
These files are the configuration files for the formatter. Don't remove!